enhancements/sig-policy/54-core-policy-module/README.md

```diff
@@ -0,0 +1,157 @@
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+A common repository / module will contain common API validation, structs, and
+methods that multiple policy controllers can use. This will simplify creating
+new controllers, and assist in maintenance of existing ones. 
+
+## Motivation
+
+Current policy controllers rely on detailed knowledge from the policy framework 
+implementation, that is not clear to new users. Most notably, the format needed
+for events to be picked up by the policy framework is not well documented. Other
+details will be more consistent if maintained in a single place.
+
+### Goals
+
+1. Common API validation: fields like `.spec.severity` which use enums will have
+consistent validation through kubebuilder when custom policy types embed them.
+1. Required API features: fields like `.spec.remediationAction` are required on
+all policy types so they can be passed down from the parent policy. Putting
+these in a struct that policy types embed will ensure they are implemented.
+1. Common methods: behavior like selecting namespaces and emitting compliance
+events for the policy framework will have a common implementation. This will
+reduce bugs from inconsistency, and allow policy controllers to focus on their
+specific behavior, not the details of interacting with the framework.
+
+### Non-Goals
+
+1. Custom Controller Tutorial: it will be easier to create custom controllers,
+but it is not a requirement that we maintain a tutorial around this process.
+1. Custom Controller Template: we will not maintain a template/sdk like the 
+operator-sdk for new policy controllers.
+
+## Proposal
+
+Create a new repository in open-cluster-mangement-io to provide implementations
+of common APIs and functions for controllers to work with the policy framework.
+
+### User Stories
+
+#### Story 1
+
+As a developer of a policy controller, I can ensure consistent behavior with
+other policies by incorporating the common API types.
+
+#### Story 2
+
+As a creator of a new policy controller, I can focus on my controller's 
+specialized detection of violations, and rely on common implementations of
+functions in order to interact with the policy framework.
+
+#### Story 3
+
+As a developer of multiple policy controllers, I can fix a bug or add a feature
+to common policy behavior in one place, and not duplicate it in each controller.
+
+### Implementation Details/Notes/Constraints 
+
+Although kubebuilder will automatically pull API validation from embedded
+structs (like when types embed `metav1.TypeMeta`), it will not automatically
+pull RBAC permissions on methods into the project. So RBAC requirements on
+common functions (eg `get namespaces` for selecting namespaces) will need to
+be well-documented.
+
+### Risks and Mitigation
+
+(unknown)
+
+## Design Details
+
```

Comment:
I think it needs more details on what you would like to put in this module. Some examples will be helpful